### PR TITLE
Overriding the getPublications for realtime.service

### PR DESCRIPTION
**Pull request by jiwalker-usgs**

### CODE REVIEW

**Comment by jiwalker-usgs** [comment]
@jread-usgs one for you?

File: R/realtime.service.R [hunk 1]
```diff
@@ -102,8 +102,8 @@ realtime.service = function(serviceEndpoint,attribute=NULL){
       tiny.text <- paste("Modeled", gsub("mean", "Mean", gsub("extreme", "Extreme", tiny.text)))
     }
     
-    full.publications  <-	getPublications(doc)
-    keywords	<-	getKeywords (doc,subType)
+    full.publications <- getPublications_(doc)
+    keywords <- getKeywords (doc,subType)
```
Comment (jread-usgs) [inline R/realtime.service.R]: :)

File: R/realtime.service.R [hunk 2]
```diff
@@ -140,6 +140,21 @@ extractCollectionDate = function(doc, attr) {
...
 }
 
+getPublications_ <- function(doc){
+  # get all linked publications
+  onlinks	<- list()
+  onlinks$data <- list()
+  onlinks$publications	<- list()
+  onlinks$resources	<- c(getNodeSet(doc,'//crossref/citeinfo/onlink'),getNodeSet(doc,'//srccite/citeinfo/onlink'), getNodeSet(doc,'//lworkcit/citeinfo/onlink'))
+  publications <- list(data=list(),publications=list(),resources=list())
+  names	<- NULL
```
Comment (jread-usgs) [inline R/realtime.service.R]: is `names` used anywhere?

Comment (jread-usgs) [inline R/realtime.service.R]: nevermind. I see.

**Comment by jread-usgs** [comment]
@jiwalker-usgs is this new? https://travis-ci.org/USGS-R/hazardItems/builds/67540860#L1580

**Comment by jiwalker-usgs** [comment]
Yeah, I haven't seen that before, checking on CSW now.

**Comment by jiwalker-usgs** [comment]
http://cida.usgs.gov/coastalhazards/csw/?service=CSW&request=GetRecordById&version=2.0.2&typeNames=fgdc:metadata&id=urn:uuid:12d6c1ee-a9e6-11e3-99f6-0050569524e0&outputSchema=http://www.opengis.net/cat/csw/csdgm&elementSetName=full

works for me, not sure why Travis is having a problem.

**Pull request merged**